package.json

```diff
@@ -29,29 +33,30 @@
     "gulp-rename": "1.2.2",
     "gulp-sass": "2.3.2",
     "gulp-sourcemaps": "1.6.0",
+    "gulp-streamify": "1.0.2",
     "gulp-template": "4.0.0",
+    "gulp-uglify": "3.0.0",
     "gulp-wrap": "0.11.0",
     "gulp-zip": "3.1.0",
     "http-response-object": "1.1.0",
     "listdirs": "3.0.0",
-    "lodash": "4.16.4",
+    "lodash": "4.17.11",
     "merge-stream": "1.0.0",
     "minimist": "1.2.0",
     "prompt": "1.0.0",
+    "protractor": "5.0.0",
+    "protractor-image-comparison": "1.2.3",
     "request": "2.72.0",
-    "require-dir": "0.3.0",
+    "require-dir": "0.3.2",
     "run-sequence": "1.2.1",
     "shelljs": "0.6.0",
+    "stream-to-promise": "^2.2.0",
     "tar-fs": "1.13.0",
-    "protractor": "5.0.0",
-    "fs-extra": "2.1.2",
-    "protractor-image-comparison": "1.2.3"
+    "vinyl-buffer": "1.0.0",
+    "vinyl-source-stream": "1.1.0"
   },
   "engines": {
     "node": ">=4.2"
   },
-  "repository": "https://github.com/ExLibrisGroup/primo-explore-devenv",
-  "dependencies": {
```

Comment:
This all looks good. We just want to make sure we make a mental note that the we've removed these dependency lines in case it breaks something.

Reply:
Looks good and as far as I can tell we should be a-ok but mentally keeping that note about the dependency lines.